docs/spl-docs/timechart-command.md

```diff
@@ -0,0 +1,105 @@
+# timechart
+
+## Description
+
+Creates a time series chart with corresponding table of statistics.
+
+A timechart is a statistical aggregation applied to a field to produce a chart, with time used as the X-axis. You can specify a split-by field, where each distinct value of the split-by field becomes a series in the chart. With the `limit` you can specify series filtering. If you set limit=0, no series filtering occurs.
+
+## Syntax
+
+The required syntax is in **bold**.
+
+timechart \
+[limit=\<chart-limit-opt>] \
+[\<bin-options>... ] \
+(**\<single-agg>** [BY **\<split-by-clause>**] )
+
+## Required Arguments
+
+When specifying `timechart` command arguments, `<single-agg>`  BY `<split-by-clause>` is required.
+
+### single-agg
+
+**Syntax:** `count or \<stats-func>(\<field>)` \
+**Description:** A single aggregation applied to a single field, including an evaluated field. For \<stats-func>, see [Stats function options](link-for-stats-page.md). No wildcards are allowed. The field must be specified, except when using the count function, which applies to events as a whole.
```

Comment:
On line 25 I think `link-for-stats-page.md` is a TODO since that page isn't merged in yet. So can you mark it as TODO so it's easier for us to find later (ideally in a way that doesn't render)? Having `todo` in the link name would work; also I think we've used this syntax for comments and it doesn't render:
```
<!-- TODO: once all docs are merged we can add links to the pages -->
```